### DISCUSSION TITLE
The fans?

### DISCUSSION THREAD

**Discussion by Avpman2** [xblax/flashforge_ad5m_klipper_mod#185] [Q&A]
I have read more than one post here about issues with the fans in the ADM5 Pro—something about pins being reversed, etc. Whatever it is/was, has it been fixed in the most recent .05 Beta? If not fixed, is there something I need to do on my end? FYI, I print a lot of ABS, and I'm counting on the filtration of the ADM 5 Pro.

Thanks again for a GREAT mod.

**Comment by consp** [comment]
The pins are not reversed, but there is a chance the internal servo duct for the internal/external selection is reversed (or incorrectly installed). If you switch to internal/external or vice versa, and have the fans off, and it sounds like something is strangling a tiny electric motor than you have that problem.

See #68 

If you have that problem, also check it in the flashforge firmware. In my case it only worked correctly in the original firmware after I moved the duct door.
